src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -263,7 +263,7 @@ public function chown($files, $user, bool $recursive = false)
      *
      * @throws IOException When the change fails
      */
-    public function chgrp($files, $group, bool $recursive = false)
+    public function chgrp(string|iterable $files, string|int $group, bool $recursive = false)
     {
         foreach ($this->toIterable($files) as $file) {
```

Comment:
The `toIterable()` method can also get a union type (`toIterable(string|iterable $files)`).